docs/meshcloud.policies.md

```diff
@@ -5,7 +5,7 @@ title: meshPolicies
 
 ## Introduction: what are meshPolicies?
 
-A meshPolicy is a set of rule(s) between two [meshObjects](meshcloud.index.md#introduction) that are globally defined by your organization. Each rule describes what [tag](meshcloud.metadata-tags.md) values have to be present on both meshObjects, in order to comply with the meshPolicy. meshPolicies are enforced in various places when attaching meshObjects. How meshPolicies can be configured is described [here](administration.mesh-policies.md).
+A meshPolicy describes a restriction between two [meshObjects](meshcloud.index.md#introduction) that are globally defined by your organization. Each meshPolicy describes what [tag](meshcloud.metadata-tags.md) values have to be present on both meshObjects, in order to comply with the meshPolicy. meshPolicies are enforced in various places when attaching meshObjects. How meshPolicies can be configured is described [here](administration.mesh-policies.md).
```

Comment:
I would really like the documentation and also the panel at some point reflect to two scopes of a policy. 1. the meshobject scope and 2. the tag scope. Instead of appear lets right "will be shown". Lets also write a sentence that the policy will protect the compliance of users, projects and environments in a multi-cloud setup. This is basically why we do this.

Reply:
We've discussed how we can improve the understanding. We'll add within an additional PR an diagram/image to improve the understanding.